_config.yml

```diff
@@ -17,7 +17,6 @@ title: Status - Ideas
 email: support@status.im
 description: >- # this means to ignore newlines until "baseurl:"
   Swarm Home. New, completed and in-progress features for Status.
-baseurl: "/ideas" # the subpath of your site, e.g. /blog
 url: "https://status-im.github.io/" # the base hostname & protocol for your site, e.g. http://example.com
```

Comment:
Shouldn't this change to https://ideas.status.im/? Maybe I'm misunderstanding.

Reply:
can be implicit I think, but url should also be removed (sloppy local test).